docs/rules/object-curly-newline.md

```diff
@@ -322,82 +314,12 @@ let {k = function() {
 }} = obj;
 ```
 
-### multiline and minProperties
-
-Examples of **incorrect** code for this rule with the `{"multiline": true, "minProperties": 2}` option:
-
-```js
-/*eslint object-curly-newline: ["error", {"multiline": true, "minProperties": 2}]*/
-/*eslint-env es6*/
-
-let a = {
-};
-let b = {
-    foo: 1
-};
-let c = {foo: 1, bar: 2};
-let d = {foo: 1,
-    bar: 2};
-let e = {foo: function() {
-    dosomething();
-}};
-
-let {
-} = obj;
-let {
-    f
-} = obj;
-let {g, h} = obj;
-let {i,
-    j} = obj;
-let {k = function() {
-    dosomething();
-}} = obj;
-```
-
-Examples of **correct** code for this rule with the `{"multiline": true, "minProperties": 2}` option:
-
-```js
-/*eslint object-curly-newline: ["error", {"multiline": true, "minProperties": 2}]*/
-/*eslint-env es6*/
-
-let a = {};
-let b = {foo: 1};
-let c = {
-    foo: 1, bar: 2
-};
-let d = {
-    foo: 1,
-    bar: 2
-};
-let e = {
-    foo: function() {
-        dosomething();
-    }
-};
-
-let {} = obj;
-let {f} = obj;
-let {
-    g, h
-} = obj;
-let {
-    i,
-    j
-} = obj;
-let {
-    k = function() {
-        dosomething();
-    }
-} = obj;
-```
-
 ### separating configuration
```

Comment:
An alternative to consider for this heading is **ObjectExpression and ObjectPattern**